package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "5.2.21",
+  "version": "5.3.0",
```

Comment:
Def go up a minor or major version. I think we're already at 5.3.x.

Reply:
Copy that, thanks!